emsdk.py

```diff
@@ -1912,10 +1918,9 @@ def install_tool(self):
 
     # Sanity check that the installation succeeded, and if so, remove unneeded
     # leftover installation files.
-    if self.is_installed():
+    if self.is_installed(assume_is_installed_version=True):
```

Comment:
How about calling this argument "skip_version_check" ?

Reply:
Good idea, renamed.